pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -758,32 +804,44 @@ pub mod pallet {
 				Self::deposit_event(Event::RefundComplete { id: pool_id });
 			}
 
-			Ok(())
+			Ok(Some(T::WeightInfo::refund_account(currency_count.to_owned())).into())
 		}
 
 		#[pallet::call_index(11)]
 		#[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
```

Comment:
Weight not updated.

Reply:
Ops: [7334d3b](https://github.com/KILTprotocol/kilt-node/pull/789/commits/7334d3b916518aba2cc9f4b7b474ed90665ecc07)